app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less

```diff
@@ -113,7 +113,7 @@
 
         .page-main & {
             .block {
-                margin-bottom: 0;
+                margin-bottom: 20px;
```

Comment:
Please use `@indent__base` variable instead.

Reply:
Thank you, it is done.